thrift/storypackage.thrift

```diff
@@ -0,0 +1,10 @@
+
+struct StoryPackageItem {
+  1: required string internalPageCode,
```

Comment:
The commas are indeed optional (and I hate all optional token delimiters).

Should we use them or not?